### PR TITLE
fix: enable and fix browser tests

### PR DESCRIPTION
**Pull request by abetaev**
[related issue](https://github.com/libp2p/js-libp2p-kad-dht/issues/89)

i wasn't able to figure out why `rpc/index.ts` test fails in browser

### CODE REVIEW

File: src/query-self.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { setMaxListeners } from 'events'
+import { setMaxListeners } from './utils.js'
```
Comment (mpetrunic) [inline src/query-self.ts:1]: why? events should work in browser

Comment (abetaev) [reply src/query-self.ts:1]: maybe it will with [this package](https://www.npmjs.com/package/events) in dependencies.

should i update the PR?

Comment (mpetrunic) [reply src/query-self.ts:1]: yeah, lets try that instead

Comment (abetaev) [reply src/query-self.ts]: i have updated. 

the `npm run test:node`runs tests ending with `.node.ts`, but `npm test` does not. is it the way it supposed to work?

Comment (mpetrunic) [reply src/query-self.ts]: Are you sure, they are both same command?
```
    "test": "aegir test -t node",
    "test:node": "aegir test -t node",
```

**Review by mpetrunic — APPROVED** [review]

**Review by wemeetagain — APPROVED** [review]
thanks @abetaev

**Pull request merged**